df2gspread/gfiles.py

```diff
@@ -77,7 +77,8 @@ def get_worksheet(gc, gfile_id, wks_name, write_access=False):
             wks = spsh.worksheet(wks_name)
         else:
             if write_access == True:
-                wks = spsh.add_worksheet(wks_name, 1000, 100)
+                rows, cols = new_sheet_dimensions
+                wks = spsh.add_worksheet(wks_name, rows, cols)
```

Comment:
This part 
```
rows, cols = new_sheet_dimensions
wks = spsh.add_worksheet(wks_name, rows, cols)
```
can be replased with this line
`spsh.add_worksheet(wks_name, *new_sheet_dimensions)`

Yes, checking of size of new_sheet_dimensions..